concurrency/pom.xml

```diff
@@ -0,0 +1,391 @@
+                        <descriptor>src/assembly/assembly-server-tck-lib.xml</descriptor>
+                    </descriptors>
+                    <finalName>${jboss.server.tck-lib}</finalName>
+                    <appendAssemblyId>false</appendAssemblyId>
+                </configuration>
+                <executions>
+                    <execution>
+                        <id>assembly-server-tck-libs</id>
+                        <phase>test-compile</phase>
+                        <goals>
+                            <goal>single</goal>
+                        </goals>
+                    </execution>
+                </executions>
+            </plugin>
+            <!-- end::testServerDep[] -->
+            <!-- Compile plugin for any supplemental class files -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>${maven.comp.plugin.version}</version>
+                <configuration>
+                    <source>${maven.compiler.source}</source>
+                    <target>${maven.compiler.target}</target>
+                </configuration>
+            </plugin>
+            <!-- tag::configTestNG[] -->
+            <!-- Surefire plugin - Entrypoint for TestNG -->
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-surefire-plugin</artifactId>
+                <version>${maven.surefire.plugin.version}</version>
+                <configuration>
+                    <!-- tag::logging[] -->
+                    <systemProperties>
+                        <property>
+                            <name>java.util.logging.config.file</name>
+                            <value>${logging.config}</value>
+                        </property>
+                    </systemProperties>
+                    <!-- end::logging[] -->
+                    <!-- tag::ignore[] -->
+                    <systemPropertyVariables>
+                        <!-- Properties shared with Arquillian -->
+                        <tck_server>${jboss.server.name}</tck_server>
+                        <tck_hostname>localhost</tck_hostname>
+                        <tck_username>tck_username</tck_username>
+                        <tck_password>tck_password</tck_password>
+                        <tck_port>8080</tck_port>
+                        <tck_port_secure>8443</tck_port_secure>
+                    </systemPropertyVariables>
+                    <!-- end::ignore[] -->
+                    <suiteXmlFiles>
+                        <suiteXmlFile>${suiteXmlFile}</suiteXmlFile>
+                    </suiteXmlFiles>
+                    <!-- Ensure surfire plugin looks under src/main/java instead of src/test/java -->
+                    <testSourceDirectory>${basedir}${file.separarator}src${file.separarator}main${file.separarator}java${file.separarator}</testSourceDirectory>
```

Comment:
This is the default, I don't think we need it defined.

Reply:
default is src/test/java, but there are no test sources at all so I removed it anyway